app/forms/two_factor_authentication/webauthn_delete_form.rb

```diff
@@ -34,6 +35,10 @@ def configuration
 
     private
 
+    attr_reader :skip_multiple_mfa_validation
+
+    alias_method :skip_multiple_mfa_validation?, :skip_multiple_mfa_validation
```

Comment:
IMO it's clearer to have one manually defined method than two auto-genned ones
```suggestion
    def skip_multiple_mfa_validation?
      !!@skip_multiple_mfa_validation
    end
```

Reply:
Hm, I kinda like the `attr_reader` approach, though I do wish it were possible to alias in one go (would love something like `attr_reader :skip_multiple_mfa_validation, as: :skip_multiple_mfa_validation?`).

I recently discovered [Ruby 3 "endless method" syntax](https://learn.shortruby.com/blog/history-of-endless-method), maybe this is a good use-case?

```suggestion
    def skip_multiple_mfa_validation? = @skip_multiple_mfa_validation
```

Reply:
I'm not used to the endless methods yet so if it were up to me, I'd stick with my suggestion! But even the PR as-is is probably fine, this is not that crucial to me